roles/config-rh-sso/tasks/setup-rh-sso.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: 'Deploy Red Hat Single Sign-On'
+  import_role:
+    name: ../roles/config-packages
+
+- name: 'Update SSO bind addresses'
```

Comment:
Could you use a wildcard (i.e.: `*`) here instead of requiring the server IPs?

Reply:
With the intent to make the service to listen to all interfaces? No. According to the documentation, an IP address must be specified.
Nevertheless, I tested it with a wildcard and, although the rh-sso7 service comes up, I am unable to access the console via the web browser.

Reply:
Update: Although I can't use the `*` wildcard, I can use `0.0.0.0` instead, which I am testing right now if it doesn't interferes with the other components.